monwork.js

```diff
@@ -0,0 +1,30 @@
+const taskcluster = require('taskcluster-client');
```

Comment:
Whoops :p

Reply:
Haha, `git add -a` is my nemesis.  At least I didn't have creds in that file :)